ClimsoftVer4/ClimsoftVer4/ucrFormDaily2.vb

```diff
@@ -282,16 +283,20 @@ Public Class ucrFormDaily2
                 If elemTotal = expectedTotal Then
                     bValueCorrect = True
                 Else
-                    MessageBox.Show("Value in [Total] textbox is different from that calculated by computer! " & elemTotal, "Error in total", MessageBoxButtons.OK, MessageBoxIcon.Error)
-                    ucrInputTotal.SetBackColor(Color.Cyan)
-                    bValueCorrect = False
-                End If
-                bValueCorrect = (elemTotal = expectedTotal)
-                If Not bValueCorrect Then
                     MessageBox.Show("Value in [Total] textbox is different from that calculated by computer! The computed total is " & elemTotal, "Error in total", MessageBoxButtons.OK, MessageBoxIcon.Error)
-                    ucrInputTotal.SetBackColor(Color.Cyan)
+                    ucrInputTotal.SetBackColor(Color.Red)
+                    ucrInputTotal.GetFocus()
+                    bValueCorrect = False
                 End If
 
+                'bValueCorrect = (elemTotal = expectedTotal)
```

Comment:
This check is similar to the above one, ant thats why we were having same message box appearing twice. which was very annoying. 
So i have removed that now to work properly. You can have a check and test

Reply:
Yep. I had noticed a similar problem also